code/style.css

```diff
@@ -1,4 +1,158 @@
 html {
-  background: #0025ff;
-  color: #fff;
+    background: #e0e2f1;
+    color: rgb(27, 26, 26);
+}
+
+body {
+    margin: 0;
+    box-sizing: border-box;
+}
+
+.header {
+    position: fixed;
+    top: 0;
+    width: 100%;
+    background-color: rgb(60, 136, 73);
+}
+
+.nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    margin: auto;
+}
+
+    .nav ul {
+        list-style: none;
+        display: flex;
+    }
+        .nav ul li a {
+            text-decoration: none;
+            font-family: 'Rajdhani', sans-serif;
+            font-size: 36px;
+            color: whitesmoke;
+            padding: 1.5rem;
+            transition: all .7s ease;
+        }
+
+            .nav ul li a:hover {
+                color: rgb(241, 177, 124);
+            }
+
+    .nav img {
+        width: 75px;
+        margin-top: 10px;
+        margin-left: 10px;
+        height: auto;
+    }
+
+
+h1 {
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+h2 {
+    font-family: 'Rajdhani', sans-serif;
+}
+
+p {
+    font-family: 'Scope One', serif;
+}
+
+.container {
+    margin-top: 115px;
+    margin-bottom: 50px;
+    padding: 2rem;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-gap: 1.5rem;
+}
+
+.image {
```

Comment:
You have achieved the responsiveness, great! I noticed that the pictures are changing dimension as if they are not set to cover (even though they are), weird. I tried to make the picture more responsive but failed. When I made the picture more responsive, something else stopped working.. Sorry I could not of been to more help.